noir-projects/aztec-nr/aztec/src/utils/bytes.nr

```diff
@@ -1,9 +1,17 @@
+// This file contains some unused functions that may be needed later (e.g. those filling top 6 bits of a 31-byte
+// field). Little endian functions are kept for reference but likely won't be used. Unused functions should be
+// removed before audit.
+
+pub mod array;
+pub mod bounded_vec;
+
 use crate::oracle::random::random;
```

Comment:
I felt like the comments here needed updating. Note that I plan to move contents of this file to bytes/array.nr once I get a review. Didn't move this file straight away to have a guarantee that the comments diff is nicely displayed and to not clutter the PR with imports updates which are largely uninteresting. Would also like to get a feedback whether the reviewer is fine with the new structure of having bytes/array.nr and bytes/bounded_vec.nr.